cmd/geth/config.go

```diff
@@ -153,6 +161,10 @@ func makeFullNode(ctx *cli.Context) *node.Node {
 
 	utils.RegisterEthService(stack, &cfg.Eth)
 
+	if ctx.GlobalBool(utils.DashboardEnabledFlag.Name) {
+		utils.RegisterDashboardService(stack, &cfg.Dashboard)
+	}
+
```

Comment:
Style nitpick. The closing `}` already defines an empty line, so there's little value in adding yet another empty line after. It just makes the code fall apart. I'd recommend not adding extra empty lines in these cases unless it's really important that the reader sees something completely different is coming up.